### PR TITLE
Remove deprecated commands/Commands/FrameChange/PersistentValue

### PR DESCRIPTION
**Pull request by lcapelluto**
### Summary

Remove deprecated commands! 🙌

### CODE REVIEW

**Comment by lcapelluto** [comment]
I don't know what's going on. Somehow the docs are taking over an hour to build, so the build gets cancelled. Not sure what would cause it to hang like that.

**Comment by mtreinish** [comment]
> I don't know what's going on. Somehow the docs are taking over an hour to build, so the build gets cancelled. Not sure what would cause it to hang like that.

This is something I noticed yesterday (or earlier this week the days have been blending together for me lately), it looks like since we enabled parallel sphinx builds in ci with #4477 we've triggered a bug in jupyter execute (which I still need to reproduce and report upstream). Basically it looks like when a jupyter cell exits with an error/exception the parallel worker stops without reporting it's done to the parent process, so the sphinx build sits waiting for a failed worker process to finish forever. Well until we hit the azure pipeline timeout and the job gets cancelled. I'd suggest trying the docs build locally (probably removing `-j auto` from the tox.ini so it doesn't sit waiting locally too) to test and find the error, or just relaunch the job and watch the progress before it gets cancelled, it should print out the error.

My guess based on this PR is that there is a jupyter execute somewhere in the docs or release notes that's using the deprecated things being removed here that's crashing the docs builds.

**Comment by SooluThomas** [comment]
@mtreinish Is using GitHub Actions for docs build a good option here?

**Comment by mtreinish** [comment]
> @mtreinish Is using GitHub Actions for docs build a good option here?

That doesn't really solve the problem here, it would actually make it worse since the timeout is 6 hours instead of 1 hour so we'd be eating up a job for much longer. The parallel docs jobs for terra typically take about 12-13 min which is far from being in danger of hitting a timeout. The only reason we're having an issue here now is because of a bug in the jupyter-sphinx extension with parallel execution. We could turn off parallel execution and docs builds would go back up to 15-20min running serially but we'd avoid the bug where a failure in `jupyter-execute` results in a job being cancelled because of a timeout. But, we switched to parallel sphinx builds to get a faster feedback time on CI, at 20mins for docs we'd typically be > 30min for full results on an idle pipeline and much longer when there are lots of patches in flight, so I'm not sure if we want to do that.

File: releasenotes/notes/remove-deprecated-pulse-commands-18381c59264090dd.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    Terra release 0.13.0 and have now been removed. You will have to upgrade
+    your Pulse code if you were still using commands. For example::
+
+        # old
+        Command(args)(channel)
+
+        # new
+        Instruction(args, channel)
+
+    ::
```
Comment (taalexander) [inline releasenotes/notes/remove-deprecated-pulse-commands-18381c59264090dd.yaml:14]: This seems odd, but maybe I don't know enough about what it is doing.

Comment (lcapelluto) [reply releasenotes/notes/remove-deprecated-pulse-commands-18381c59264090dd.yaml:14]: I wanted to make two separate literal blocks

**Pull request merged**